{"name":"test_verifyTypeToException","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"exception message mismatch expected [true] but found [false]","trace":"java.lang.AssertionError: exception message mismatch expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:96)\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\n\tat org.testng.Assert.assertTrue(Assert.java:44)\n\tat test.java.tests.test09_verifyTypeUnheldException.test_verifyTypeToException(test09_verifyTypeUnheldException.java:25)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\n\tat java.base/java.lang.Thread.run(Thread.java:832)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1626982350077,"stop":1626982363329,"uuid":"3f272891-119f-415d-8ac9-bcb3b95f84a3","historyId":"659ed6e76f8a75f1cc22d05ab336fec2","fullName":"test.java.tests.test09_verifyTypeUnheldException.test_verifyTypeToException","labels":[{"name":"package","value":"test.java.tests.test09_verifyTypeUnheldException"},{"name":"testClass","value":"test.java.tests.test09_verifyTypeUnheldException"},{"name":"testMethod","value":"test_verifyTypeToException"},{"name":"parentSuite","value":"Appium Demo Test Suite"},{"name":"suite","value":"Appium Demo Test"},{"name":"subSuite","value":"test.java.tests.test09_verifyTypeUnheldException"},{"name":"host","value":"FVFY7177HV22"},{"name":"thread","value":"41160@FVFY7177HV22.TestNG-test=Appium Demo Test-1(16)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}